source/Editor.js

```diff
@@ -1613,20 +1613,37 @@ var addLinks = function ( frag, root, self ) {
     while ( node = walker.nextNode() ) {
         data = node.data;
         parent = node.parentNode;
-        while ( match = linkRegExp.exec( data ) ) {
+
+        while ( match = linkRegExp.exec( data )) {
             index = match.index;
             endIndex = index + match[0].length;
             if ( index ) {
                 child = doc.createTextNode( data.slice( 0, index ) );
                 parent.insertBefore( child, node );
             }
+
+            var href;
+            var link = match[1];
+            var email = match[2];
+            var networkPath = match[3];
+
+            if (link){
+                if ( /^(?:ht|f)tps?:/.test( link )) {
+                   href = link;
+                }
+                else {
+                    href = 'http://' + link;
+                }
+            } else if ( email ) {
+                href = 'mailto:' + email;
+            } else if ( self._config.linkifyNetworkPaths && networkPath ) {
```

Comment:
If the configuration flag is set to false, this will still create an anchor, but with an undefined href. We need to check the configuration before matching against the regex.